lib/Hakyll/Web/Feed.hs

```diff
@@ -154,7 +184,21 @@ renderAtomWithTemplates ::
     -> [Item String]           -- ^ Feed items
     -> Compiler (Item String)  -- ^ Resulting feed
 renderAtomWithTemplates feedTemplate itemTemplate config context = renderFeed
-    feedTemplate itemTemplate config
+    XmlFeed feedTemplate itemTemplate config
+    (makeItemContext "%Y-%m-%dT%H:%M:%SZ" context)
+
+
+--------------------------------------------------------------------------------
+-- | Render a JSON feed using given templates with a number of items.
+renderJsonFeedWithTemplates ::
+       Template                -- ^ Feed template
+    -> Template                -- ^ Item template
+    -> FeedConfiguration       -- ^ Feed configuration
+    -> Context String          -- ^ Item context
+    -> [Item String]           -- ^ Feed items
+    -> Compiler (Item String)  -- ^ Resulting feed
+renderJsonFeedWithTemplates feedTemplate itemTemplate config context = renderFeed
```

Comment:
For consistency with other functions, this should be renamed to `renderJsonWithTemplates` and added to the exports list.